libs/api-client-tiger/src/axios.ts

```diff
@@ -22,6 +22,9 @@ const _CONFIG: AxiosRequestConfig = {
         put: {
             "Content-Type": "application/json;charset=utf8",
         },
+        get: {
+            Accept: "application/vnd.gooddata.api+json",
```

Comment:
Is this safe? Can we instead use the default headers specified in https://github.com/gooddata/gooddata-ui-sdk/pull/970/files#diff-72216b0b9af677699864dac77f0165de1c8efd82a99101f3eaf8d23c5f6fddc7R84

Reply:
I mean changing the default for all the get requests can yield unexpected results.

Reply:
makes sense, changed it